sleap/gui/dialogs/importvideos.py

```diff
@@ -200,6 +206,18 @@ def __init__(
             button_layout.addWidget(all_channels_first_button)
             all_channels_last_button.clicked.connect(self.set_all_channels_last)
             all_channels_first_button.clicked.connect(self.set_all_channels_first)
+        if any(
+            [
+                widget.import_type["video_type"] == "single_image"
+                for widget in self.import_widgets
+            ]
+        ):
+            all_grayscale_button = QPushButton("All grayscale")
+            all_rgb_button = QPushButton("All RGB")
+            button_layout.addWidget(all_grayscale_button)
+            button_layout.addWidget(all_rgb_button)
+            all_grayscale_button.clicked.connect(self.set_all_grayscale)
+            all_rgb_button.clicked.connect(self.set_all_rgb)
```

Comment:
We can add a test for this in the tests/gui/test_import.py/test_gui_import function using the jpeg test images we already have.
https://github.com/talmolab/sleap/blob/57ee08a7490aa950f61b14f80fa8e2d372a2ca5c/tests/gui/test_import.py#L6-L35